catalog/ui/catalog-ui-search/src/main/webapp/component/session-timeout-modal/session-timeout.controller.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+/* jshint browser: true */
+/* global define */
+define(['jquery',
+        'underscore',
+        'marionette',
+        'js/wreqr',
+        './lightbox/lightbox.view',
+        './session-timeout.view'
+    ], function ($, _, Marionette, wreqr, BlockingLightbox, SessionTimeoutModal) {
+        'use strict';
+        var SessionTimeoutController;
+
+        var _this = null;
```

Comment:
What is this variable being used for?

Reply:
It's no longer being used. It can be removed.